source/common/common/utility.cc

```diff
@@ -102,6 +102,30 @@ std::string StringUtil::subspan(const std::string& source, size_t start, size_t
   return source.substr(start, end - start);
 }
 
+std::vector<std::string> StringUtil::tokenize(const std::string& source,
+                                              const std::string& splitters) {
+  bool charmap[256] = {false};
+  for (size_t i = 0; i < splitters.size(); i++) {
+    charmap[static_cast<uint8_t>(splitters[i])] = true;
+  }
+  size_t strlen = source.size();
```

Comment:
It might be less scary not to use the same name as the C library function whose name shall never be spoken in polite company.